extensions/amp-animation/0.1/runners/native-web-animation-runner.js

```diff
@@ -77,18 +77,20 @@ export class NativeWebAnimationRunner extends AnimationRunner {
    */
   init() {
     devAssert(!this.players_);
-    this.players_ = this.requests_.map(request => {
-      // Apply vars.
-      if (request.vars) {
-        setStyles(request.target, assertDoesNotContainDisplay(request.vars));
+    this.players_ = this.requests_.map(
+      /** @type {../web-animation-types.InternalWebAnimationRequestDef} */ request => {
+        // Apply vars.
+        if (request.vars) {
+          setStyles(request.target, assertDoesNotContainDisplay(request.vars));
+        }
+        const player = request.target.animate(
+          /** @type {!Array<Object>} */ (request.keyframes),
+          request.timing
+        );
```

Comment:
`request.timing` matches https://developer.mozilla.org/en-US/docs/Web/API/Element/animate, so it's safe to cast to a `KeyframeAnimationOptions`

Reply:
Fixed both typedefs with the respective native types.

Reply:
The typecast should be on `request.timing` only, not where it is currently.

Reply:
Good catch. Fixed.